blender/arm/props_ui.py

```diff
@@ -2303,6 +2308,70 @@ def draw(self, context):
         if n != None and (n.bl_idname == 'ShaderNodeRGB' or n.bl_idname == 'ShaderNodeValue' or n.bl_idname == 'ShaderNodeTexImage'):
             layout.prop(context.active_node, 'arm_material_param')
 
+class ARM_OT_CopyToBundled(bpy.types.Operator):
+    bl_label = 'Copy To Bundled'
+    bl_idname = 'arm.copy_to_bundled'
+    bl_description = ('Copies and repaths external image assets to project Bundled folder')
+
+    def execute(self, context):
+        self.copy_to_bundled(bpy.data.images)
+        return {'FINISHED'}
+
+    @classmethod
+    def copy_to_bundled(self, data):
+        wrd = bpy.data.worlds['Arm']
+        project_path = arm.utils.get_fp()
+
+        # Blend - Images
+        for asset in data:
+            if asset.filepath_from_user() != '':
+                bundled_filepath = project_path + '/Bundled/' + asset.name
+                try:
+                    # Exists -> Yes
+                    if os.path.isfile(bundled_filepath):
+                        # Override - > Yes
+                        if (wrd.arm_copy_override):
+                            if asset.has_data:
+                                asset.filepath_raw = bundled_filepath
+                                asset.save()
+                                asset.reload()
+                                # Syntax - Yellow
+                                print(log.colorize(f'Asset name "{asset.name}" already exists, overriding the original', 33), file=sys.stderr)
+                            else:
+                                # Syntax - Red 
+                                log.error(f'Asset name "{asset.name}" has no data to save or copy, skipping')
+                                continue
+                        # Override - > No
+                        else:
+                            # Syntax - Yellow
+                            print(log.colorize(f'Asset name "{asset.name}" already exists, skipping', 33), file=sys.stderr)
+                            continue
+                    # Exists -> No
+                    else:
+                        if os.path.isfile(bundled_filepath):
```

Comment:
What's the point of this if statement? The entire else block around that is only executed if `bundled_filepath` does not exist, so it looks like this would always evaluate to false (assuming that no other process creates the file in between, which would "fail" with this code anyways).

Reply:
Not sure why I added that. Removed it, thanks.